src/boot/efi/boot.c

```diff
@@ -740,17 +739,14 @@ static BOOLEAN menu_run(
 
                 case KEYPRESS(0, SCAN_F1, 0):
                 case KEYPRESS(0, 0, 'h'):
+                case KEYPRESS(0, 0, 'H'):
                 case KEYPRESS(0, 0, '?'):
                         /* This must stay below 80 characters! */
-                        status = StrDuplicate(L"(d)efault (t/T)timeout (e)dit (v)ersion (Q)uit (r)esolution (P)rint (h)elp");
-                        break;
-
-                case KEYPRESS(0, 0, 'Q'):
-                        exit = TRUE;
-                        run = FALSE;
```

Comment:
please don't drop this. this i useful for debugging things from the efi cmdline.